sources/Shaarei_Orah/Shaarei_Orah_index_post.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+import sys
+import os
+p = os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+sys.path.insert(0, p)
+from sources.local_settings import *
+sys.path.insert(0, SEFARIA_PROJECT_PATH)
+os.environ['DJANGO_SETTINGS_MODULE'] = "local_settings"
+from sources.functions import *
+import re
+import data_utilities
+import codecs
+from sources import functions
+from Shaarei_Orah_parse import *
```

Comment:
Be careful with `import *`. Generally I would recommend against creating a separate script for index creation and text posting.